source/Octopus.Client.E2ETests/Octopus.Client.E2ETests.csproj

```diff
@@ -25,9 +25,10 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-        <PackageReference Include="NSubstitute" Version="4.3.0" />
-        <PackageReference Include="NUnit" Version="3.8.1" />
-        <PackageReference Include="NUnit3TestAdapter" Version="3.8.0" />
+        <PackageReference Include="NSubstitute" Version="4.4.0" />
+        <PackageReference Include="NUnit" Version="3.13.3" />
+        <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
+        <PackageReference Include="System.IO.Compression" Version="4.3.0" />
```

Comment:
Was the dependency on `System.IO.Compression` added intentionally?

Reply:
Yes, the newer version of `NUnit3TestAdapter` takes a dependency on it, and it's not built into the BCL in NETFRAMEWORK